src/Microsoft.Win32.Registry.AccessControl/tests/Configurations.props

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp-Windows_NT;
-      netfx-Windows_NT;
```

Comment:
Don't we still want to test against netfx? Harvesting, doesn't mean we no longer support this OOB package in netfx, we're still shipping a netfx asset as part of the package.

Reply:
Right we are still shipping a netfx asset. But that asset is produced in a release branch, not in master. We will probably infinitely ship a netfx asset. Not removing the live build / live test hinders us making progress.

Reply:
Should we then, remove all framework specific tests to full framework in registry in favor of this PR? Or at least call it out on the issue Dan opened that community will be helping to remove those guys 😄

Reply:
There is only very few which I just removed.